<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandeWidget</class>
 <widget class="QWidget" name="CommandeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <addaction name="actionNew"/>
     <addaction name="actionEdit"/>
     <addaction name="actionDelete"/>
     <addaction name="separator"/>
     <addaction name="actionEnvoyerEmail"/>
     <addaction name="separator"/>
     <addaction name="actionExportPDF"/>
     <addaction name="actionRefresh"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxSearch">
         <property name="title">
          <string>Recherche et Filtres</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
                 <item>
                  <widget class="QLabel" name="labelSearch">
                   <property name="text">
                    <string>Rechercher :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditSearch">
                   <property name="placeholderText">
                    <string>ID, libellé, fournisseur…</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QLabel" name="labelFournisseur">
                   <property name="text">
                    <string>Fournisseur :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboFournisseur">
                   <item>
                    <property name="text">
                     <string>Toutes</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Fournisseur A</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Fournisseur B</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Fournisseur C</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Fournisseur D</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Autres</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QLabel" name="labelStatus">
                   <property name="text">
                    <string>Statut :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboStatus">
                   <item>
                    <property name="text">
                     <string>Tous</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>En cours</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Validée</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Prête</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Annulée</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QLabel" name="labelDateFrom">
                   <property name="text">
                    <string>Date depuis :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="dateFrom">
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QLabel" name="labelDateTo">
                   <property name="text">
                    <string>Jusqu'à :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="dateTo">
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableCommandes">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Libellé</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fournisseur</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>État</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Date Arrivée</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Date Sortie</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Montant Total</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="rightPanel">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QGroupBox" name="groupBoxActions">
              <property name="title">
               <string>Actions Rapides</string>
              </property>
              <layout class="QVBoxLayout">
               <item>
                <widget class="QPushButton" name="btnNewCommande">
                 <property name="text">
                                                 <string>Créer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnEditCommande">
                 <property name="text">
                                                 <string>Modifier</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnDeleteCommande">
                 <property name="text">
                                                 <string>Annuler / Supprimer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnEnvoyerEmail">
                 <property name="text">
                                                 <string>Envoyer Email</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxQuickInfo">
              <property name="title">
               <string>Infos Rapides</string>
              </property>
              <layout class="QVBoxLayout">
               <item>
                <widget class="QLabel" name="labelEtat">
                 <property name="text">
                  <string>État :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="quickEtat">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelFournisseur">
                 <property name="text">
                  <string>Fournisseur :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="quickFournisseur">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDateArrivee">
                 <property name="text">
                  <string>Date Arrivée :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="quickDateArrivee">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDateSortie">
                 <property name="text">
                  <string>Date Sortie :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="quickDateSortie">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelMontant">
                 <property name="text">
                  <string>Montant Total :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="quickMontant">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
  </widget>
 <resources/>
 <connections/>
</ui>
